### PR TITLE
Add `right-click` delete function

### PR DESCRIPTION
**Pull request by joennlae**
Thanks for this excellent tool for measuring my upcoming apartment's floor plan. 

I added a delete line function when right-clicking on a line and formatted it a bit :-) But nice to find a nearly 10-year-old to fix my little problem :-).

### CODE REVIEW

File: js/augenmass-main.js [hunk 1]
```diff
@@ -48,207 +48,225 @@ var end_bracket_len = 5;
...
-	    return;
-	getView().drawAll();
+  this.onClick = function (e, x, y) {
+    if (e.which != undefined && e.which == 3) {
+      // right mouse button.
+      if (getModel().hasEditLine()){
+        cancelCurrentLine();
+      } else{
+        // delete line when not editing
+        console.log("we are here", e, getModel().findClosest(x, y))
```
Comment (hzeller) [inline js/augenmass-main.js]: This looks like some leftover debug message ...

**Review by hzeller — CHANGES_REQUESTED** [review]
Thanks!

If possible, can you squash that together to two commits to clearly show the to operations, the stylle and formatting cleanup and the feature adding ?

  * Adding the feature and updating the README
  * Doing the formatting updates

(or the other way around, up to you).

**Comment by hzeller** [comment]
Fixes issues #1

**Comment by joennlae** [comment]
Thank you for the review :-) I adopted your proposed changes. 

Have a good one. Cheers :-)

**Comment by hzeller** [comment]
Thanks, merged!

**Pull request merged**